go-controller/pkg/ovn/ovn.go

```diff
@@ -544,6 +552,27 @@ func (oc *Controller) WatchNetworkPolicy() error {
 	return err
 }
 
+// WatchEgressFirewall starts the watching of egressfirewall resource and calls
+// back the appropriate handler logic
+func (oc *Controller) WatchEgressFirewall() error {
+	_, err := oc.watchFactory.AddEgressFirewallHandler(cache.ResourceEventHandlerFuncs{
+		AddFunc: func(obj interface{}) {
+			egressFirewall := obj.(*egressfirewall.EgressFirewall)
+			oc.addEgressFirewall(egressFirewall)
```

Comment:
I prefer having each individual `Add`/`Update`/`Delete` handler return an error and have that being logged here.

Reply:
done